### PR TITLE
Photo uploading UI

### PR DESCRIPTION
**Pull request by sunmengyue**
### What this PR does (required):
- I created the UI to upload a profile photo

### Screenshots / Videos (front-end only):
- 
Screen Shot 2022-01-19 at 9 36 11 PM

### Any information needed to test this feature (required):
- Need to login as a test user

### CODE REVIEW

**Comment by sundayezeilo** [comment]
Please link the ticket for this PR

**Comment by sunmengyue** [comment]
The ticket is #24  https://github.com/hatchways/team-bamboo/issues/24
Thanks!

File: client/src/components/ProfilePhoto/ProfilePhoto.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+      <SettingHeader header="Profile Photo" />
+      <Avatar
+        alt="profile photo"
+        src={`https://www.nj.com/resizer/zovGSasCaR41h_yUGYHXbVTQW2A=/1280x0/smart/cloudfront-us-east-1.images.arcpublishing.com/advancelocal/SJGKVE5UNVESVCW7BBOHKQCZVE.jpg`}
+        sx={{ width: 120, height: 120, marginBottom: 4 }}
+      />
+      <Box textAlign="center">
+        <Typography sx={{ fontSize: 15, fontWeight: 600, color: '#afafaf' }}>
```
Comment (rajivtitus) [inline client/src/components/ProfilePhoto/ProfilePhoto.tsx]: Let's consider using a `Typography` variant for font size (we can set custom font sizes for variants in MUI's theme) and also use the theme to apply font color

Ditto where applicable

File: client/src/components/ProfilePhoto/ProfilePhoto.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        </Typography>
+        <form>
+          <Input id="button-file" name="images" type="file" onChange={handleImgChange} sx={{ opacity: 0 }} />
+          <InputLabel htmlFor="button-file">
+            <Button component="span" variant="outlined" color="primary" size="large" sx={{ px: 1.5, py: 2 }}>
+              Upload a file from your device
+            </Button>
+          </InputLabel>
+        </form>
+        <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center', mt: 3 }}>
```
Comment (rajivtitus) [inline client/src/components/ProfilePhoto/ProfilePhoto.tsx]: We can minimize this code by using MUI's `Button` component

```
<Button variant="text" startIcon={<DeleteIcon />}>
  Delete
</Button>
```

**Review by rajivtitus — COMMENTED** [review]
UI looks good! 👍🏽 
Just a couple of comments for you to address

**Review by rajivtitus — APPROVED** [review]
LGTM! Thanks for making those changes 👍🏽

**Pull request merged**